Card/src/main/java/com/paypal/android/card/CardClient.kt

```diff
@@ -1,6 +1,20 @@
 package com.paypal.android.card
 
-import com.paypal.android.core.PaymentsClient
+import com.paypal.android.card.network.CardAPIClient
+import com.paypal.android.core.APIClient
+import com.paypal.android.core.PaymentsConfiguration
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 
-@Suppress("UnusedPrivateMember")
-class CardClient(paymentsClient: PaymentsClient)
+class CardClient(val configuration: PaymentsConfiguration) {
```

Comment:
Question - do we want to add doc strings as we go for this? I don't want to cause a bunch of rework since these classes are still in flux, but could be good to do it now to make it clear which classes / methods / fields will be merchant facing and which stuff is internal

Reply:
That's a good callout. We could also start declaring classes `internal` where appropriate. I think `CardClient` and whatever is in its public interface will be merchant facing though.